src/main/scala/com/wavesplatform/settings/fee/OrderFeeSettings.scala

```diff
@@ -0,0 +1,129 @@
+package com.wavesplatform.settings.fee
+
+import cats.data.Validated
+import cats.implicits._
+import com.wavesplatform.settings.fee.AssetType.AssetType
+import com.wavesplatform.settings.fee.Mode.Mode
+import com.wavesplatform.transaction.AssetId
+import com.wavesplatform.transaction.assets.exchange.AssetPair
+import monix.eval.Coeval
+import net.ceedubs.ficus.Ficus._
+import net.ceedubs.ficus.readers.EnumerationReader._
+import net.ceedubs.ficus.readers.ValueReader
+import play.api.libs.json.{JsObject, Json}
+
+import scala.util.Try
+
+object OrderFeeSettings {
+
+  type ErrorsListOr[A] = Validated[List[String], A]
+
+  sealed trait OrderFeeSettings {
+
+    /** Returns json for order fee settings taking into account fee that should be paid for matcher's account script invocation */
+    def getOrderFeeJson(minMarcherFee: Long): Coeval[JsObject] = Coeval.evalOnce {
```

Comment:
Just `json` as other entities has?

Reply:
Which one?
Orders and transactions have `Coeval[JsObject]` type of `json` field

Reply:
I meant, may be we use the same name :) We are in OrderFeeSettings, `orderFee` here is unnecessary

Reply:
Done